tests/test_files/pineko.toml

```diff
@@ -0,0 +1,13 @@
+[paths]
+# inputs
+ymldb = "data/yamldb"
```

Comment:
you can even skip the `data` prefix, I'd say ...

Reply:
Yes but I don't know at the moment if I will need some other files to be put in `test_files`. In that case I would prefer to keep them separate from the data